lib_xud/doc/rst/version.rst

```diff
@@ -0,0 +1 @@
+.. version:: 2.0.2
```

Comment:
Please change to 1.2.0 so the data sheet version matches the release version.  I know it's all a bit of a nonsense at the moment because we haven't verified which App Notes actually work with this release of lib_xud, but version 2.0.2 doesn't even exist.

Reply:
It should be fixed now